### PR TITLE
Matching game header component #20

### PR DESCRIPTION
**Pull request by ssinikoglu**
Please review. This PR consists of Image and words matching game header component. I will also make other PR's for image word matching game components and container. Test fails because the component is not meaningful alone.

### CODE REVIEW

File: src/components/MatchingGameHeader/index.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <>
+      <h2 className="matchingGame__title">{t("games.matchingGame.title")}</h2>
+      <p className="matchingGame__context">
+        {t("games.matchingGame.subtitle")}
+      </p>
+    </>
+  );
+};
+
```
Comment (ibrahimaljarad) [inline src/components/MatchingGameHeader/index.jsx]: hey sule can you delete th extra newlines

Comment (ssinikoglu) [reply src/components/MatchingGameHeader/index.jsx]: Thanks for reviewing Ibrahim. Ihave made the change that you have requested. PTAL

**Review by louisrli — APPROVED** [review]

**Review by ibrahimaljarad — APPROVED** [review]

**Comment by ssinikoglu** [comment]
PR is approved. PTAL @Ammar-64

**Comment by Ammar-64** [comment]
> PR is approved. PTAL @Ammar-64 

@ssinikoglu can you please update the branch, pull the dev branch, merge it with yours and push again

**Comment by ssinikoglu** [comment]
> > PR is approved. PTAL @Ammar-64
> 
> @ssinikoglu can you please update the branch, pull the dev branch, merge it with yours and push again

Thanks @Ammar-64 ! I have just updated the branch and pushed to Github.

**Comment by ssinikoglu** [comment]
> > > PR is approved. PTAL @Ammar-64
> > 
> > 
> > @ssinikoglu can you please update the branch, pull the dev branch, merge it with yours and push again
> 
> Thanks @Ammar-64 ! I have just updated the branch and pushed to Github.

I have updated and add Arabic translations

> > > PR is approved. PTAL @Ammar-64
> > 
> > 
> > @ssinikoglu can you please update the branch, pull the dev branch, merge it with yours and push again
> 
> Thanks @Ammar-64 ! I have just updated the branch and pushed to Github.

I have updated branch also added Arabic translations @Ammar-64, PTAL.

**Pull request merged**